### PR TITLE
feature/tpi-mobile-sectors-page

### PR DESCRIPTION
**Pull request by batraisk**

### CODE REVIEW

File: app/javascript/components/charts/bubble/MobileChart.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            </div>
+            <div className="item-body">
+              <hr />
+              {activeSector[el].length === 0 && <div className="no-companies">No companies</div>}
+              {activeSector[el].length > 0 && (
+                <ul className="companies-list">
+                  {activeSector[el].map((company, index) => (
+                    <li
+                      key={`chart-companies-${index}`}
+                      onClick={() => { window.location.href = `tpi/companies/${company.slug}`; }}
```
Comment (simaob) [inline app/javascript/components/charts/bubble/MobileChart.js]: I think it should be -> `/tpi/companies/${company.slug}`

File: app/assets/stylesheets/tpi/_charts.scss [hunk 2]
```diff
@@ -96,3 +96,4 @@ $scrollbar-track-border: $neutral-color-5 !default;
 @import 'charts/cp-all-sectors';
 @import 'charts/mq-sector-pie-chart';
 @import 'charts/mq-level';
+@import 'charts/mobile-booble-chart';
```
Comment (tsubik) [inline app/assets/stylesheets/tpi/_charts.scss]: typo, should be `bubble`

File: app/javascript/components/charts/bubble/MobileChart.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        components={{DropdownIndicator}}
+        theme={customSelectTheme}
+      />
+
+      <div className="accordions-list">
+        {levelsSignature.map((el, i) => (
+          <div key={`chart-item-list-${i}`} className={`item-list level-item-color-${el} ${openItems.includes(i) ? ' open' : ''}`}>
+            <div className="item-header" onClick={() => setOpenItemByIndex(i)}>
+              <div className="item-title">
+                <div>livel {levelsSignature[el]}</div>
```
Comment (tsubik) [inline app/javascript/components/charts/bubble/MobileChart.js]: Typo `level`

**Comment by tsubik** [comment]
:eyes:

**Comment by tsubik** [comment]
Also, the company selector could be improved. Font size, and indent level.
image

File: app/assets/stylesheets/tpi/_charts.scss [hunk 4]
```diff
@@ -96,4 +96,4 @@ $scrollbar-track-border: $neutral-color-5 !default;
 @import 'charts/cp-all-sectors';
 @import 'charts/mq-sector-pie-chart';
 @import 'charts/mq-level';
-@import 'charts/mobile-booble-chart';
+@import 'charts/mobile-buble-chart';
```
Comment (tsubik) [inline app/assets/stylesheets/tpi/_charts.scss]: still typo `buble` -> `bubble` ;)

Comment (batraisk) [reply app/assets/stylesheets/tpi/_charts.scss]: oh, sorry)

**Comment by batraisk** [comment]
fixed)

**Comment by tsubik** [comment]
Looks like I did not remove all rubocop disable Metrics. You can remove them @batraisk to make the CI green.

**Comment by tsubik** [comment]
Almost there :) 

Go to sector has some wrong styles.

image

File: app/javascript/components/charts/bubble/MobileChart.js [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+};
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <img src={props.selectProps.menuIsOpen ? chevronUpIconBlack : chevronDownIconBlack} />
+  </components.DropdownIndicator>
+);
+
+
+const MobileBubbleChart = ({ levels }) => {
```
Comment (tsubik) [inline app/javascript/components/charts/bubble/MobileChart.js]: Hmm, I just realized, that this is not a chart but more like CompaniesAccordion or MobileSectorsOverview duno. Maybe it is better to rename that component?

**Comment by batraisk** [comment]
done)

**Pull request merged**